src/fragments/orders.ts

```diff
@@ -78,12 +78,25 @@ export const fragmentOrderLine = gql`
   fragment OrderLineFragment on OrderLine {
     id
     isShippingRequired
+    allocations {
+      quantity
+      warehouse {
+        id
+      }
+    }
     variant {
       id
       quantityAvailable
       preorder {
         endDate
       }
+      stocks {
```

Comment:
We should add ids by default for caching purposes. Can we add the id to stocks and allocations?

Reply:
sure!